### ISSUE TITLE
Document iOS Build Settings For Quiet Audio Issue

### ISSUE THREAD

**Issue by briveramelo** [RenderHeads/UnityPlugin-AVProMovieCapture#330]
**Is your feature request related to a problem? Please describe.**
Volume playback was very very quiet on iOS.

**Describe the solution you'd like**
I found that two settings should be checked in the iOS build settings to resolve this, as noted in this other seemingly unrelated issue:
https://github.com/Placeholder-Software/Dissonance/issues/131

Player Settings > iOS > Other Settings
1. Prepare iOS for Recording
2. Force iOS Speakers when Recording

When I applied these iOS build settings, volume playback was perfect. 

Making a note of this in the documentation would have saved me some time and may save others some time too. Consider including this in the iOS Platform section.

Thanks for making this great tool ❤️

**Comment by Chris-RH** [comment]
Thank you, we will look into updating our documentation. :)

**Comment by Chris-RH** [comment]
Did you try using SetMicrophoneRecordingHint ?
https://www.renderheads.com/content/docs/AVProMovieCapture/api/RenderHeads.Media.AVProMovieCapture.NativePlugin.html#RenderHeads_Media_AVProMovieCapture_NativePlugin_SetMicrophoneRecordingHint_System_Boolean_RenderHeads_Media_AVProMovieCapture_NativePlugin_MicrophoneRecordingOptions_

**Comment by briveramelo** [comment]
Yes. I run through this in a setup IEnumerator whenever the component is enabled with this snippet:
```
if (!Application.HasUserAuthorization(UserAuthorization.WebCam))
{
  yield return Application.RequestUserAuthorization(UserAuthorization.WebCam);
}

if (!Application.HasUserAuthorization(UserAuthorization.Microphone))
{
  yield return Application.RequestUserAuthorization(UserAuthorization.Microphone);
}
if (Application.HasUserAuthorization(UserAuthorization.Microphone))
{
  NativePlugin.SetMicrophoneRecordingHint(true);
}
```

**Comment by MorrisRH** [comment]
The following matches enabling `Force iOS Speakers when Recording` and would be the preferred way to do it:
```csharp
Native.SetMicrophoneRecordingHint(true, NativePlugin.MicrophoneRecordingOptions.DefaultToSpeaker);
```
